recipe/meta.yaml

```
@@ -1,13 +1,13 @@
 {% set name = "Flask-Caching" %}
-{% set version = "2.1.0" %}
+{% set version = "2.3.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: b7500c145135836a952e3de3a80881d9654e327a29c852c9265607f5c449235c
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/flask_caching-{{ version }}.tar.gz
```

Please make this small modification:

```suggestion
  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_').lower() }}-{{ version }}.tar.gz
```